src/types/Role.res

```diff
@@ -0,0 +1,67 @@
+type t = [
```

Comment:
I guess role props is not the same as Accessibility.role ?

Reply:
Yes, they are not the same. At first I merged them together but then I found out they are not the same and reverted the change.